.github/workflows/e2e-tests-atomic.yml

```diff
@@ -18,20 +18,16 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  # Run tests against WP Nightly & WC latest. Only one combination at a time.
```

Comment:
I think this was copied - we don't test the WP version or WC versions on Atomic. In fact, the Atomic site is already set up for us.